tests/fixtures/setup_py/package_with_funtion_call_as_name/setup.py

```diff
@@ -0,0 +1,23 @@
+from setuptools import find_packages, setup
+
+
+def find_name():
+    return "foo"
```

Comment:
the issue is this will actually resolve the name, but the package in question won't resolve to any name (the one that opens a file to determine its name) and therefore wont wind up in the lockfile -- we will need to actually run its setup file to determine its name -- so the change actually has to happen to `setup_info.py` in the ast parser.